testdata/p4_16_samples/table-entries-exact-bmv2.p4

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
You should also write some tests where you use "default" to match a tuple, and also where actions have data-plane arguments.